Example/Backpack/View Controllers/BPKRootListTableViewController.m

```diff
@@ -35,8 +35,9 @@ - (void)viewDidLoad {
     UIImage *largeSettingsIcon = [BPKIcon templateIconNamed:@"settings" size:BPKIconSizeLarge];
     [self.settingsButton setImage:largeSettingsIcon];
```

Comment:
This should be removed no?

Reply:
Do you mean for it to look like this?
https://github.com/Skyscanner/backpack-ios/pull/178/commits/9238f3170600cb0a7eef9e962725fb4c183a561d